mitosheet/mitosheet/public/v3/sheet_functions/bool_functions.py

```diff
@@ -123,6 +123,52 @@ def IF(condition: pd.Series, true_series: AnyPrimitiveOrSeriesInputType, false_s
     )
 
 
+@cast_values_in_arg_to_type('argv', 'bool')
+@handle_sheet_function_errors
+def IFS(*argv: Optional[IfsInputType]) -> pd.Series:
+    """
+    {
+        "function": "IFS",
+        "description": "Returns one value if the condition is True. Returns the other value if the conditon is False.",
+        "search_terms": ["ifs", "if", "conditional", "and", "or"],
+        "examples": [
+            "IFS(Status == 'success', 1, 0)",
+            "IFS(Nums > 100, 100, Nums)",
+            "IFS(AND(Grade >= .6, Status == 'active'), 'pass', 'fail')"
+        ],
+        "syntax": "IFS(boolean_condition_1, value_if_true, boolean_condition_2, value_if_true)",
+        "syntax_elements": [{
+                "element": "boolean_condition",
+                "description": "An expression or series that returns True or False values. Valid conditions for comparison include ==, !=, >, <, >=, <=."
+            },
+            {
+                "element": "value_if_true",
+                "description": "The value the function returns if condition is True."
+            }
+        ]
+    }
+    """
+    # Go through all of the conditions. The first condition that is true for
+    # *each cell* should be selected. 
+    results = pd.Series()
+    for index, condition in enumerate(argv):
+        # The args are alternating between the conditions and the values corresponding to them.
+        # If this is a series, we assume it is the condition. We skip the values because they're
+        #   only used as a function of the conditions. 
```

Comment:
I don't think we should do this! I think that even numbered args should be treated as conditions (if possible), and odd number args should be treated as results. 

This means the code below needs to figure out if the true_series is really a series or if it might be just a constant.